deebot_client/commands/xml/clean.py

```diff
@@ -0,0 +1,27 @@
+"""Clean commands."""
+
+from __future__ import annotations
+
+from deebot_client.models import CleanAction, CleanMode
+
+from .common import ExecuteCommand
+
+
+class CleanArea(ExecuteCommand):
+    """Clean area command."""
+
+    NAME = "Clean"
+    HAS_SUB_ELEMENT = True
+
+    def __init__(self, mode: CleanMode, area: str, cleanings: int = 1) -> None:
+        # <ctl><clean type='SpotArea' act='s' speed='standard' deep='1' mid='4,5'/></ctl>
+
+        super().__init__(
+            {
+                "type": mode.xml_value,
+                "act": CleanAction.START.xml_value,
+                "speed": "standard",  # TODO: FanSpeedLevel.NORMAL.xml_value, after #560 is merged
```

Comment:
Would this mean that the bot will always start with the fan level at `standard` and not the one defined in the fan speed entity?

Reply:
I think it makes sense to pass it as parameter. But we have to wait for #560 for the new FanSpeedLevel Enum.

We can either merge it like it is and address this in another PR, or we wait for #560